code/datums/components/crafting/crafting.dm

```diff
@@ -432,12 +449,15 @@
 	var/req_text = ""
 	var/tool_text = ""
 	var/catalyst_text = ""
+	var/machinery_text = ""
 
 	for(var/a in R.reqs)
 		//We just need the name, so cheat-typecast to /atom for speed (even tho Reagents are /datum they DO have a "name" var)
 		//Also these are typepaths so sadly we can't just do "[a]"
 		var/atom/A = a
 		req_text += " [R.reqs[A]] [initial(A.name)],"
+	for(var/atom/movable/content as anything in R.machinery)
```

Comment:
```suggestion
	for(var/obj/machinery/content as anything in R.machinery)
```
Just so we are clear here, these paths should normally be `/obj/machinery` by default, no?
This won't have any effects on the code, but should be more informative to the reader.

Question, however: is this another case of path-or-instance? If so then it should be clarified.
And if this is necessarily a type path or an instance, then saying so would be good.

Reply:
Yes, I don't think it's a case of path or instance, isn't it just used to index the list?